src/plugins/interactive_setup/server/kibana_config_writer.ts

```diff
@@ -152,6 +180,28 @@ export class KibanaConfigWriter {
     return { raw: rawConfig, parsed: parsedConfig };
   }
 
+  /**
+   * Build config for Fleet outputs
+   * @param caCert
+   * @param host
+   */
+  private static getFleetDefaultOutputConfig(caCert: string, host: string): FleetOutputConfig[] {
+    const cert = new X509Certificate(caCert);
+    const certFingerprint = cert.fingerprint256;
```

Comment:
Here the fingerprint is in hex format something like `A6:8E:63:D8:73:C5:BA:CA:FC:0F:E4:39:91:65:CA:60:29:A9:EC:E5:20:D5:AA:67:5C:01:A6:0F:30:EA:74:AA` does the agent will require a different format?

Reply:
Hex is the format we agreed upon, this LGTM 👍